gameobjects/ball.cpp

```diff
@@ -85,12 +83,19 @@ void Ball::ResetBall()
 {
     mSpeed = BALL_START_SPEED;
 
-    const float yStartingPos = distYPos(generator);
+    const float yStartingPos = Random::Range(-Y_STARTING_POSITION_BOUNDS, Y_STARTING_POSITION_BOUNDS);
     SetPosition(glm::vec3(0.0f, yStartingPos, 0.0f));
 
-    const float xDir = distStartDir(generator) ? 1.0f : -1.0f;
-    const float yDir = distStartDir(generator) ? 1.0f : -1.0f;
+    const float xDir = Random::Bool() ? 1.0f : -1.0f;
```

Comment:
I think there is a way to do a random choice in the STL, you could investigate

In python, it would be something like:

```
random.choice([1, -1])
```

Reply:
alternatively, could be a good addition to your Random class, taking in any collection of any size and randomly selecting one out of it - fun engineering subproblem there in generics programming